lldb/source/API/SBValueList.cpp

```diff
@@ -22,13 +22,14 @@ class ValueListImpl {
 public:
   ValueListImpl() = default;
 
-  ValueListImpl(const ValueListImpl &rhs) = default;
+  ValueListImpl(const ValueListImpl &rhs)
```

Comment:
This may be an ABI break if compilers generate different code or symbols for this constructor.

Reply:
Isn't `ValueListImpl` an implementation detail?

Reply:
Actually you're right here, ValueListImpl is an implementation detail and shouldn't matter. This one is fine.